marconi-chain-index/src/Marconi/ChainIndex/Orphans.hs

```diff
@@ -137,6 +137,9 @@ mapLeft _ (Right v) = Right v
 instance ToJSON C.ScriptData where
     toJSON v = Aeson.String $ Text.decodeLatin1 $ Base16.encode $ C.serialiseToCBOR v
 
+instance SQL.FromRow Int where
+  fromRow = SQL.field
```

Comment:
I wouldn't add that instance. If your select query just returns a single element which is an int, then the type would simply be `[[Int]]`, right? There is already a `FromField Int` instance, so let's use that.

Reply:
Yes, I forgot to remove it. I had added in when I needed to use the [Exist](https://sqlite.org/lang_expr.html) operator to see if an Spent existed.